setup.py

```diff
@@ -36,6 +36,7 @@
         'wrapt',
         'pytz>=2018.4',
         'tzlocal>=1.5.1',
+        'pandas>=1.1.3',
```

Comment:
Would it be possible to lower this requirement to something like `pandas>=1.0.0`? Or is `1.1.3` introducing some special feature that you use?